warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -892,17 +892,17 @@ public String getProjectFieldsAsString() {
         return StringUtils.join(this.getProjectFields(), Constants.PARAM_VALUE_SEP);
     }
 
-    public Set<String> getBlacklistedFields() {
+    public Set<String> getExcludedFields() {
         return blacklistedFields;
```

Comment:
This could be updated to `excludedFields`

Reply:
good catch. done